cavity_status.csv

```diff
@@ -1,297 +1,297 @@
+15,4,Cold landing,-24080000,126542,Online
+15,5,Cold landing,15022000,80046,Online
+15,6,Cold landing,-30879000,160292,Online
+15,7,Cold landing,-38557000,202915,Online
+15,8,Cold landing,-26101000,132075,Online
+16,1,Cold landing,-18305316,98950,Online
+16,2,Cold landing,-13015000,71711,Online
+16,3,Cold landing,-22450000,121535,Online
+16,4,Cold landing,-10289000,55638,Online
+16,5,Cold landing,-22110000,120140,Online
+16,6,Cold landing,-27605000,151619,Online
+16,7,Cold landing,-22769000,124613,Online
+16,8,Cold landing,-13555000,73790,Offline
+17,1,Cold landing,-7765000,42427,Online
+17,2,Cold landing,-11688000,64178,Online
+17,3,Cold landing,-4371000,23805,Online
+17,4,Cold landing,-9447000,52014,Online
+17,5,Cold landing,-2936000,15956,Online
+17,6,Cold landing,-19273000,105071,Online
+17,7,Cold landing,-25300000,139582,Maintenance Done
+17,8,Cold landing,-23631000,130406,Maintenance Done
+18,1,Cold landing,-33663000,177228,Online
+18,2,Cold landing,-34307000,184158,Online
+18,3,Cold landing,-48268000,256282,Online
+18,4,Cold landing,-48410000,263771,Online
+18,5,Cold landing,-47238000,256621,Online
+18,6,Cold landing,-41883420,224969,Online
+18,7,Cold landing,-35474000,189575,Online
+18,8,Cold landing,-23704000,128709,Online
+19,1,Cold landing,-21613000,116731,Online
+19,2,Cold landing,-20703000,112058,Online
+19,3,Cold landing,-18680000,101161,Online
+19,4,Cold landing,-9295000,39149,Online
+19,5,Cold landing,-13116500,71777,Online
+19,6,Cold landing,-21861500,118645,Online
+19,7,Cold landing,-15140000,81074,Online
+19,8,Cold landing,-20982000,112840,Online
+20,1,Cold landing,-17455000,93724,Online
+20,2,Cold landing,-5155000,27820,Online
+20,3,Cold landing,-16620000,90085,Online
+20,4,Cold landing,-25830000,140482,Online
+20,5,Cold landing,-18643500,101966,Online
+20,6,Cold landing,-27246000,147955,Online
+20,7,Cold landing,-16203000,88098,Online
+20,8,Cold landing,-15079000,80457,Online
+21,1,Cold landing,-17827000,96419,Online
+21,2,Cold landing,-20437000,111521,Online
+21,3,Cold landing,-10378000,56786,Online
+21,4,Cold landing,-20580000,113409,Online
+21,5,Cold landing,-17125000,94194,Online
+21,6,Cold landing,-22363000,122207,Online
+21,7,Cold landing,-15855000,87140,Online
+21,8,Cold landing,-14805000,80645,Online
+22,1,Cold landing,-19661500,106680,Online
+22,2,Cold landing,-26137000,139982,Online
+22,3,Cold landing,-24062000,128257,Online
+22,4,Cold landing,-26439000,143230,Online
+22,5,Cold landing,-7170000,38032,Online
+22,6,Cold landing,-12510000,66663,Online
+22,7,Cold landing,-12044000,65411,Online
+22,8,Cold landing,-20393000,111659,Online
+23,1,Cold landing,-20030100,109101,Online
+23,2,Cold landing,-22401000,123382,Online
+23,3,Cold landing,-20345000,111952,Online
+23,4,Cold landing,-17015000,93400,Online
+23,5,Cold landing,-19707000,107935,Online
+23,6,Cold landing,-20522000,112373,Online
+23,7,Cold landing,-13547000,74863,Online
+23,8,Cold landing,0,0,Offline
+24,1,On resonance,-23890000,129662,Online

```

Comment:
This one was not set to 'cold landing'. I fixed it now. Please run and upload again